exercise/staff_views.py

```diff
@@ -530,3 +532,61 @@ def form_valid(self, form):
     def form_invalid(self, form):
         messages.error(self.request, _('FAILURE_SAVING_CHANGES'))
         return super().form_invalid(form)
+
+class SubmissionConversion(SubmissionMixin, BaseRedirectView):
```

Comment:
To follow the naming convention, I think the class name should be SubmissionConversionView, right?

Reply:
Ahh i see, well yes that makes sense. I will update that. thanks